Fix NoClassDefFoundError occurring upon enabling the plugin.

[02:07:43] [Server thread/INFO]: [MedievalFactions] Enabling MedievalFactions v4.6-alpha-4
[02:07:43] [Server thread/WARN]: [MedievalFactions] Loaded class preponderous.ponder.misc.Pair from DansEssentials v2.1 which is not a depend, softdepend or loadbefore of this plugin.
[02:07:43] [Server thread/ERROR]: Error occurred while enabling MedievalFactions v4.6-alpha-4 (Is it up to date?)
java.lang.NoClassDefFoundError: preponderous/ponder/misc/Savable
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1010) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:197) ~[patched_1.17.1.jar:git-Paper-399]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:586) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:108) ~[patched_1.17.1.jar:git-Paper-399]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[patched_1.17.1.jar:git-Paper-399]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	at dansplugins.factionsystem.data.PersistentData$LocalStorageService.loadPlayerActivityRecords(PersistentData.java:1421) ~[Medieval-Factions-4.6-alpha-4.jar:?]
	at dansplugins.factionsystem.data.PersistentData$LocalStorageService.load(PersistentData.java:1306) ~[Medieval-Factions-4.6-alpha-4.jar:?]
	at dansplugins.factionsystem.MedievalFactions.load(MedievalFactions.java:156) ~[Medieval-Factions-4.6-alpha-4.jar:?]
	at dansplugins.factionsystem.MedievalFactions.onEnable(MedievalFactions.java:54) ~[Medieval-Factions-4.6-alpha-4.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.17.1.jar:git-Paper-399]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.1.jar:git-Paper-399]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.1.jar:git-Paper-399]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:561) ~[patched_1.17.1.jar:git-Paper-399]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:475) ~[patched_1.17.1.jar:git-Paper-399]
	at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:733) ~[patched_1.17.1.jar:git-Paper-399]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:317) ~[patched_1.17.1.jar:git-Paper-399]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1220) ~[patched_1.17.1.jar:git-Paper-399]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-399]
	at java.lang.Thread.run(Thread.java:831) ~[?:?]
Caused by: java.lang.ClassNotFoundException: preponderous.ponder.misc.Savable
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:151) ~[patched_1.17.1.jar:git-Paper-399]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[patched_1.17.1.jar:git-Paper-399]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
	... 22 more

[07:36:52] [Server thread/ERROR]: Could not load 'plugins/Medieval-Factions-4.6-alpha-5.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: java.lang.NoClassDefFoundError: preponderous/ponder/minecraft/abs/PonderPlugin
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:157) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:414) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:322) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.CraftServer.loadPlugins(CraftServer.java:422) ~[paper-1.18.1.jar:git-Paper-175]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:285) ~[paper-1.18.1.jar:git-Paper-175]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.1.jar:git-Paper-175]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-175]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.NoClassDefFoundError: preponderous/ponder/minecraft/abs/PonderPlugin
	at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
	at java.lang.ClassLoader.defineClass(ClassLoader.java:1012) ~[?:?]
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:197) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:587) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:108) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:71) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:153) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	... 7 more

#1331 maybe fixed it but
>[19:52:12 INFO]: [MedievalFactions] Enabling MedievalFactions v4.6-alpha-5
[19:52:12 ERROR]: Error occurred while enabling MedievalFactions v4.6-alpha-5 (Is it up to date?)
java.lang.NoClassDefFoundError: preponderous/ponder/misc/Pair
        at dansplugins.factionsystem.services.LocalLocaleService.getPairFromLine(LocalLocaleService.java:195) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at dansplugins.factionsystem.services.LocalLocaleService.lambda$loadMissingKeysFromInputStream$0(LocalLocaleService.java:177) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at java.util.Iterator.forEachRemaining(Iterator.java:133) ~[?:?]
        at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845) ~[?:?]
        at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
        at dansplugins.factionsystem.services.LocalLocaleService.loadMissingKeysFromInputStream(LocalLocaleService.java:176) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at dansplugins.factionsystem.services.LocalLocaleService.loadFromResource(LocalLocaleService.java:163) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at dansplugins.factionsystem.services.LocalLocaleService.loadStrings(LocalLocaleService.java:62) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at dansplugins.factionsystem.MedievalFactions.load(MedievalFactions.java:153) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at dansplugins.factionsystem.MedievalFactions.onEnable(MedievalFactions.java:53) ~[Medieval-Factions-4.6-alpha-5.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:564) ~[paper-1.18.1.jar:git-Paper-186]
        at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:478) ~[paper-1.18.1.jar:git-Paper-186]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:727) ~[paper-1.18.1.jar:git-Paper-186]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:503) ~[paper-1.18.1.jar:git-Paper-186]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:313) ~[paper-1.18.1.jar:git-Paper-186]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.1.jar:git-Paper-186]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-186]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.ClassNotFoundException: preponderous.ponder.misc.Pair
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:151) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
        ... 21 more
[19:52:12 INFO]: [MedievalFactions] Disabling MedievalFactions v4.6-alpha-5

--- a/pom.xml
+++ b/pom.xml
@@ -6,7 +6,7 @@
 
     <groupId>dansplugins</groupId>
     <artifactId>Medieval-Factions</artifactId>
-    <version>4.6-alpha-5</version>
+    <version>4.6-alpha-6</version>
     <packaging>jar</packaging>
 
     <name>Medieval Factions</name>

--- a/pom.xml
+++ b/pom.xml
@@ -95,8 +95,7 @@
             <groupId>dansplugins</groupId>
             <artifactId>Fiefs</artifactId>
             <version>v0.8-alpha-1</version>
-            <scope>system</scope>
-            <systemPath>${project.basedir}/dependencies/Fiefs-v0.8-alpha-1.jar</systemPath>
+            <scope>compile</scope>
         </dependency>
         <dependency>
             <groupId>preponderous</groupId>

--- a/pom.xml
+++ b/pom.xml
@@ -102,8 +102,7 @@
             <groupId>preponderous</groupId>
             <artifactId>ponder</artifactId>
             <version>v0.15-alpha-2</version>
-            <scope>system</scope>
-            <systemPath>${project.basedir}/dependencies/Ponder-v0.15-alpha-2.jar</systemPath>
+            <scope>compile</scope>
         </dependency>
     </dependencies>
 </project>

